main.go

```diff
@@ -64,6 +66,26 @@ func main() {
 		Holder: *holder,
 	}
 
+	var t *template.Template
+	if *licensef != "" {
+		d, err := ioutil.ReadFile(*licensef)
+		if err != nil {
+			log.Printf("template error: %s", err)
+			os.Exit(1)
+		}
+		t, err = template.New("").Parse(string(d))
+		if err != nil {
+			log.Printf("template error: %s", err)
```

Comment:
Maybe use "license template: " or "license file: " for error prefix? "template error" is quite generic and it isn't mentioned in the `-f` arg description.